pkg/sql/catalog/catprivilege/validate.go

```diff
@@ -54,6 +55,12 @@ func allowedSuperuserPrivileges(objectNameKey catalog.NameKey) privilege.List {
 	if privs != nil {
 		return privs
 	}
+	// If the object is not a registered system table but is in the system
+	// database then return the same privileges a registered system table would
```

Comment:
nit: this is true right now since we're only doing it for one table but if for whatever reason we had to do this for another table that was `ReadOnly` this would be inaccurate.

Maybe it'll be better to special case it the key lookup and do 
			`tableKey.Name = table we actually want to looukp` ?

Reply:
So if IIUC before this diff all tables in the system database that weren't registered system tables were getting `ALL` anyways so we're no worse off with this diff when it comes to `ReadOnly` tables? We could special case for just `crdb_internal_copy_tenant_settings` and `crdb_internal_copy_span_count` but that just seemed more brittle.

Reply:
Though maybe the special casing prevents all the test churn below as well 🤷‍♂️

Reply:
>So if IIUC before this diff all tables in the system database that weren't registered system tables were getting ALL anyways so we're no worse off with this diff when it comes to ReadOnly tables? We could special case for just crdb_internal_copy_tenant_settings and crdb_internal_copy_span_count but that just seemed more brittle.

Fair enough, I'm still inclined to go with the special casing for the above two and defining the two `crdb_internal_...` system tables. 

I don't think it's necessarily brittle and I'm not sure we really want the catch all case here.

Reply:
Added special case for only tables in the system database and with a table prefix equal to our sentinel prefix for a copied system table.